### DISCUSSION TITLE
FINN - How to work with ONNX)

### DISCUSSION THREAD

**Discussion by Tala22** [Xilinx/finn#899] [Q&A]
Hello,
Can any one help me please, i am new with finn, i am trying to do the basics example in (FINN - How to work with ONNX) , after Launch Jupyter notebooks (by :Launch Jupyter notebooks ,then typing : docker run -it -p 8888:8888 jupyter/scipy-notebook:2023-08-19), then i had the link to jupyter notebook like this (http://127.0.0.1:8888/lab/tree/work), i uploaded the previous example in new notebook to try , but i have the following error : no model named finn ,after execute the following command :
from finn.util.visualization import showInNetron.
and when i tried to pip install finn on terminal, the following message appeared:
requirement already satisfied: finn in /opt/conda/lib/python3.11/site-packages (0.0.1)  
any body have any idea?what is the wrong?

**Comment by fpjentzsch** [comment]
Hi,
did you follow the steps in the [Getting Started](https://finn.readthedocs.io/en/latest/getting_started.html) guide closely? You should launch the Jupyter server with `bash ./run-docker.sh notebook`.
